tests/state/conftest.py

```diff
@@ -265,11 +270,12 @@ def make_unfinished_block(parent_state,
             print('Making crosslink in shard %d' % shard)
             indices = get_crosslink_notaries(crystallized_state, shard, crosslink_shards=crosslink_shards, config=config)
             print('Indices: %r' % indices)
-            bitfield = [1 if random.random() < crosslinker_share else 0 for i in indices]
-            bitmask = bytearray((len(bitfield)+7) // 8)
-            for i, b in enumerate(bitfield):
-                bitmask[i//8] ^= (128 >> (i % 8)) * b
-            print('Bitmask:', bin(int.from_bytes(bitmask, 'big')))
+            is_voting = [random.random() < attester_share for _ in indices]
```

Comment:
Let's change this to `is_notarizing` and `notarizing` to distinguish it from the attestations above.

Reply:
Also renamed the remaining `is_voting` to `is_attesting` which seems to be the correct terminology.